### PR TITLE
Add io.github.accessory.minus_games_gui

### PR DESCRIPTION
**Pull request by Accessory**
### Please confirm your submission meets all the criteria

- [x] Please describe the application briefly.
             The Minus Games Gui offers an efficient solution for downloading, synchronizing, and
            running your games. It keeps your game files managed and up-to-date with the latest
            patches and updates from a Minus Games Server Instance. With seamless synchronization
            across devices, you can effortlessly continue your progress no matter where you are.
            This setup ensures a reliable and straightforward way to enhance and manage your gaming
            experience.

- [x] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [x] I have read the and followed all the [Submission and licence requirements][reqs].
- [x] I have [built][build] and tested the submission locally.
- [x] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about this submission. **Link:**

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

**Comment by Accessory** [comment]
bot, build io.github.accessory.minus_games_gui

**Comment by Accessory** [comment]
bot, build io.github.accessory.minus_games_gui

**Comment by Accessory** [comment]
bot, build io.github.accessory.minus_games_gui

**Comment by bbhtt** [comment]
Seems you inherited the readme contributing from master branch of this repo. can you remove them ?

File: io.github.accessory.minus_games_gui.yml [hunk 1]
```diff
@@ -0,0 +1,424 @@
...
+          stable-only: true
+          url-template: https://github.com/FeralInteractive/gamemode/releases/download/$version/gamemode-$version.tar.xz
+
+  # Minus Games Gui
+  - name: minus_games_gui
+    buildsystem: simple
+    build-options:
+      append-path: /usr/lib/sdk/rust-stable/bin
+      env:
+        "CARGO_HOME": "/run/build/minus_games_gui/cargo"
```
Comment (bbhtt) [inline io.github.accessory.minus_games_gui.yml:399]:
```suggestion
        CARGO_HOME: "/run/build/minus_games_gui/cargo"
```

Comment (Accessory) [reply io.github.accessory.minus_games_gui.yml]: done

**Comment by Accessory** [comment]
> Seems you inherited the readme contributing from master branch of this repo. can you remove them ?

deleted

**Comment by Accessory** [comment]
bot, build io.github.accessory.minus_games_gui

**Comment by Accessory** [comment]
bot, build io.github.accessory.minus_games_gui

**Comment by bbhtt** [comment]
Why does this need to be a separate package ? Can't you package both the GUI and client as one ?

**Comment by Accessory** [comment]
> Why does this need to be a separate package ? Can't you package both the GUI and client as one ?

I want to make the Client EOF after this.

**Comment by bbhtt** [comment]
Do you mean EOL ? Why ?

**Comment by Accessory** [comment]
> Do you mean EOL ? Why ?

Yes I meant EOL. And the reason is that I will most likely integrate the Client into the gui what makes the other flatpak obsolete. Furthermore, I will probably not have enough time to maintain both packages.

**Comment by bbhtt** [comment]
It's a bit problematic if these apps are such shortlived. You can EOL it but Flathub has to host the EOL-ed version for the foreseeable future which has infrastructure costs.

**Comment by bbhtt** [comment]
How does this work? I only see this screen and save/quit then it goes back to the no games found screen.

image


Where is the server part ?

**Comment by Accessory** [comment]
> It's a bit problematic if these apps are such shortlived. You can EOL it but Flathub has to host the EOL-ed version for the foreseeable future which has infrastructure costs.

It would be ok for me if the package itself gets deleted. But I could not find any way doing this.

> Where is the server part ?

The Server must be hosted separately and can be found at the github repo under Releases.  
[Repo](https://github.com/Accessory/minus_games/)
[Releases](https://github.com/Accessory/minus_games/releases)
Get Started Guide [Guide](https://github.com/Accessory/minus_games/blob/main/other/docs/quick_start.md)

**Pull request closed without merging**